### ISSUE TITLE
🐛 BUG: Wrangler login error

### ISSUE THREAD

**Issue by pepsiamir** [cloudflare/workers-sdk#4433]
### Which Cloudflare product(s) does this pertain to?

Workers Runtime

### What version(s) of the tool(s) are you using?

3.15.0

### What version of Node are you using?

18.17.1

### What operating system are you using?

Windows

### Describe the Bug

```
npx wrangler login  
 ⛅️ wrangler 3.15.0
-------------------
Attempting to login via OAuth...
Opening a link in your default browser: https://dash.cloudflare.com/oauth2/auth?response_type=code&client_id=54d11594-84e4-41aa-b438-e81b8fa78ee7&redirect_uri=http%3A%2F%2Flocalhost%3A8976%2Foauth%2Fcallback&scope=account%3Aread%20user%3Aread%20workers%3Awrite%20workers_kv%3Awrite%20workers_routes%3Awrite%20workers_scripts%3Awrite%20workers_tail%3Aread%20d1%3Awrite%20pages%3Awrite%20zone%3Aread%20ssl_certs%3Awrite%20constellation%3Awrite%20ai%3Aread%20offline_access&state=mmNc4bBFt~mFE-VgchP0e6RtJDEEuE9k&code_challenge=JjU_lWyc_pLQ9MwandRomnqPs-Ne8iAX-Ojs82hnCEk&code_challenge_method=S256
            throw a;
            ^

SyntaxError: Unexpected token < in JSON at position 0
    at JSON.parse (<anonymous>)
    at parseJSONFromBytes (D:\nodejs\d1-tutorial\node_modules\wrangler\wrangler-dist\cli.js:6582:19)
    at successSteps (D:\nodejs\d1-tutorial\node_modules\wrangler\wrangler-dist\cli.js:6553:27)
    at D:\nodejs\d1-tutorial\node_modules\wrangler\wrangler-dist\cli.js:5109:83
    at AsyncResource.runInAsyncScope (node:async_hooks:203:9)
    at AsyncResource.runMicrotask (node:internal/process/task_queues:137:8)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)

Node.js v18.17.1
```

**Comment by penalosa** [comment]
Could you provide some more details about the flow here? Did Wrangler open a web browser? Were you able to successfully log in to your Cloudflare account?

**Comment by pepsiamir** [comment]
Yes, The web browser opened and after accepting the terms, it redirected to a localhost address and I got that error.

I tried again and it worked now for some reason.

You have granted authorization to Wrangler!
Welcome to a serverless execution environment that allows you to create entirely new applications or augment existing ones without configuring or maintaining new infrastructure.

🤠[Learn more about the Wrangler CLI](https://developers.cloudflare.com/workers/cli-wrangler)
📖[Check out the Workers docs](https://developers.cloudflare.com/workers)
💡[Try out more code snippets and examples](https://developers.cloudflare.com/workers/examples)
Feel free to close this browser window.
